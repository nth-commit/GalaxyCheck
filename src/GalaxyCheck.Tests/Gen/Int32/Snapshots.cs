using Xunit;
using GC = GalaxyCheck;
using static Tests.TestUtils;

namespace Tests.Gen.Int32
{
    public class Snapshots
    {
        [Fact]
        public void Example() => SnapshotGenValues(GC.Gen.Int32());

        [Fact]
        public void ExampleOfExampleSpaces() => SnapshotGenExampleSpaces(GC.Gen.Int32().Between(-10, 10));

        [Fact]
        public void ExampleOfExampleSpaces_OriginVariant1() => SnapshotGenExampleSpaces(GC.Gen.Int32().Between(0, 10).ShrinkTowards(5));

        [Fact]
        public void ExampleOfExampleSpaces_OriginVariant2() => SnapshotGenExampleSpaces(GC.Gen.Int32().Between(-10, 0).ShrinkTowards(-5));
    }
}
