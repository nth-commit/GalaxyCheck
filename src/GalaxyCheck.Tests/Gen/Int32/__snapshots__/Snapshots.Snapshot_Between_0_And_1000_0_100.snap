[
  "1000",
  "1000",
  "1000",
  "566",
  "1000",
  "574",
  "1000",
  "555",
  "1000",
  "561",
  "121",
  "1000",
  "1000",
  "1000",
  "1000",
  "1000",
  "805",
  "1000",
  "590",
  "703",
  "824",
  "1000",
  "745",
  "119",
  "1000",
  "279",
  "497",
  "864",
  "1000",
  "1000",
  "75",
  "297",
  "764",
  "1000",
  "1000",
  "330",
  "190",
  "852",
  "1000",
  "1000",
  "931",
  "1000",
  "646",
  "428",
  "414",
  "1000",
  "816",
  "288",
  "575",
  "1000",
  "946",
  "639",
  "33",
  "1000",
  "353",
  "1000",
  "1000",
  "1000",
  "955",
  "924",
  "1000",
  "1000",
  "1000",
  "811",
  "1000",
  "938",
  "1000",
  "189",
  "5",
  "1000",
  "111",
  "823",
  "1000",
  "1000",
  "279",
  "876",
  "413",
  "1000",
  "1000",
  "333",
  "200",
  "694",
  "1000",
  "1000",
  "1000",
  "1000",
  "587",
  "1000",
  "1000",
  "1000",
  "333",
  "1000",
  "1000",
  "1000",
  "1000",
  "12",
  "1000",
  "133",
  "179",
  "1000"
]
